jazz_deployments/components/validation/common.js

```diff
@@ -72,7 +72,7 @@ var validateUnAllowedFieldsInInput = function (deployment_data, fields_list, onC
 
 var validateAllRequiredFieldsValue = function (deployment_data, required_fields, onComplete) {
     var invalid_required_fields = [];
-    _.forEach(required_fields, function (value, key) {
+    _.map(required_fields, function (value, key) {
```

Comment:
required_fields.map(...)?
Object.keys(deployment_data).length === 0 to replace _isEmpty

Reply:
Using map